specification/columns/region.md

```diff
@@ -1,20 +1,20 @@
 # Region
 
-A Region is a provider assigned identifier for an isolated geographic area where a [*resource*](#glossary:resource) is provisioned in or a [*service*](#glossary:service) is provided from. Region is commonly used for scenarios like analyzing cost and unit prices based on where *resources* are deployed.
+A Region ID is a provider assigned identifier for an isolated geographic area where a [*resource*](#glossary:resource) is provisioned in or a [*service*](#glossary:service) is provided from. Region is commonly used for scenarios like analyzing cost and unit prices based on where *resources* are deployed.
 
-Region MUST be present in the billing data and MUST be of type String. Region MUST NOT be null when a *resource* or *service* is operated in or managed from a distinct region. Region values MUST be consistent within the provider and MUST be the same values used to indicate the region when provisioning or purchasing the *resource* or *service*.
+RegionId MUST be present in the billing data and MUST be of type String. RegionId MUST NOT be null when a *resource* or *service* is operated in or managed from a distinct region. RegionId values MUST be consistent within the provider and MUST be the same values used to indicate the region when provisioning or purchasing the *resource* or *service*.
 
 ## Column ID
 
-Region
+RegionId
 
 ## Display name
 
-Region
+Region ID
 
 ## Description
 
-Isolated geographic area where a *resource* is provisioned in or a *service* is provided from.
+ID of an isolated geographic area where a *resource* is provisioned in or a *service* is provided from.
```

Comment:
In the table below, please change: "Column required | True" to "FOCUS Essential | False" similar to this pr: https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/331/files